recipes/bitmath/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "bitmath" %}
+{% set version = "1.3.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bitmath-{{ version }}.tar.gz
+  sha256: 293325f01e65defe966853111df11d39215eb705a967cb115851da8c4cfa3eb8
+
+build:
+  entry_points:
+    - bitmath = bitmath:cli_script
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
```

Comment:
I feel that there is a missing dependency on a `progressbar` called progress bar. https://github.com/tbielawa/bitmath/blob/1.3.3.1/requirements-py3.txt

or a run_constrained? Is it optional or not?

Reply:
Technically, this is optional.
https://github.com/tbielawa/bitmath/blob/19a06eceb464c83899352bc9ebfe2d5e0b9ffa22/bitmath/integrations/__init__.py#L33

However, I'm tempted to indeed add `progressbar33`, `click` and `argparse` to the run dependancies.  Those are integrations.

Reply:
If optional it's fine, just remember to add run_constrained if they ever have version requirements.

Reply:
Didn't find any trace of versions requirements. Thanks.